src/email/utils/sesTemplateEmail.ts

```diff
@@ -72,7 +72,7 @@ export const sendBulkEmails = async (data) => {
         } || {},
       ),
     })),
-    // ConfigurationSetName: 'sonu-config',
+    ConfigurationSetName: 'sonu',
```

Comment:
Here we can use our configuration set  name by which we want to track our emails. @appdatasoft